### PR TITLE
AB#8655 Reduce the size of media loaded from Content Hub ONE

### PR DESCRIPTION
**Pull request by chpa7**

### CODE REVIEW

File: play-media-native/api/queries/getMedia/index.ts [hunk 1]
```diff
@@ -14,7 +16,13 @@ query {
       fileHeight,
       fileSize,
       fileType,
-      fileUrl,
+      fileUrl (
+        transform: {
+          width: ${Math.ceil(Dimensions.get('window').width)},
+          height: ${Math.ceil(Dimensions.get('window').height / 2)},
```
Comment (sc-nelikostadinova) [inline play-media-native/api/queries/getMedia/index.ts:22]: Why half of the height? What if the image is vertical?

**Review by sc-nelikostadinova — CHANGES_REQUESTED** [review]
Media items look very blurry for me. Also I think we should do this for all images not just the ones in the media section.

[Blurry1](https://scontent.fsof11-1.fna.fbcdn.net/v/t1.15752-9/339586871_1305048620355791_5240949851941482622_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=ae9488&_nc_ohc=obFe5OQMwacAX-wgKXC&_nc_ht=scontent.fsof11-1.fna&oh=03_AdTXQ0h8aqagjzoBRaHZIpYeb1Np0xsheHNPCZ6pjqHrSA&oe=6456150B)
[OK1](https://scontent.fsof11-1.fna.fbcdn.net/v/t1.15752-9/338237587_257290433404625_7371916352545820243_n.jpg?_nc_cat=108&ccb=1-7&_nc_sid=ae9488&_nc_ohc=enYWC7Oy5PIAX_oTTXm&_nc_ht=scontent.fsof11-1.fna&oh=03_AdRXaFiy8Sye_7USURYa4ty82R_441dr9YwjmDaeN5sG1w&oe=6455FD11)

[Blurry2](https://scontent.fsof11-1.fna.fbcdn.net/v/t1.15752-9/338244328_611348401009309_620201377781613088_n.jpg?_nc_cat=104&ccb=1-7&_nc_sid=ae9488&_nc_ohc=s91laPKvoiIAX9Dx4qn&_nc_ht=scontent.fsof11-1.fna&oh=03_AdQatk71Pl0yNeUn5NkRgKsPXvYKbaZfJz-_HERvNsDOlg&oe=6455F7B4)
[OK2](https://scontent.fsof11-1.fna.fbcdn.net/v/t1.15752-9/338279756_216825571095819_7867173967765950684_n.jpg?_nc_cat=108&ccb=1-7&_nc_sid=ae9488&_nc_ohc=GtXVcMld0gkAX_mjWA3&_nc_ht=scontent.fsof11-1.fna&oh=03_AdSzZbpd8V6lUuJmwBXe-RZKwWY3p6LKal-NKqy4SElY6w&oe=6455EC12)

**Comment by chpa7** [comment]
Could you try removing the "fit: SCALEDOWN" and see if the images are less blurry for you?

For the other point, you are right, I'll add the transformations to the images in the other queries as well.

File: play-media-native/api/queries/getMedia/index.ts (hunk 1 above)
Comment (chpa7) [reply play-media-native/api/queries/getMedia/index.ts:22]: Even if the image is vertical it still can't take up full device height in grid, list or card view. For the getMediaById query it is fetched with full device height transformation.

Comment (sc-nelikostadinova) [reply play-media-native/api/queries/getMedia/index.ts:22]: Oh, makes sense!

**Review by sc-nelikostadinova — APPROVED** [review]

**Pull request merged**